extraConfigSriov.py

```diff
@@ -50,6 +50,31 @@ class ExtraConfigSriovOvSHWOL:
     def __init__(self, cc):
         self._cc = cc
 
+    def check_pci_realloc(self, client: K8sClient, mcp_name: str) -> None:
+        need_pci_realloc = False
+        for e in self._cc["workers"]:
+            ip = client.get_ip(e['name'])
+            if ip is None:
+                sys.exit(-1)
+            rh = host.RemoteHost(ip)
+            rh.ssh_connect("core")
+            if "switchdev-configuration-before-nm.service" in rh.run("systemctl list-units --state=failed --plain --no-legend").out:
+                print(f"switchdev-configuration is failing in {e['name']}, additional machine configuration is required")
+                need_pci_realloc = True
```

Comment:
`return true`

This function is not only checking if we need pci realloc.

The remainder of this function can become its own function.

Reply:
Done.